### PR TITLE
Add single sql udf page

### LINKED ISSUE #2270: add SQL UDFs to the sidebar of "User-Defined Functions"
### Related code PR

_No response_

### Which part(s) of the docs might be affected or should be updated? And how?

image

Apart from "Embedded UDFs" and "External UDFs", the sidebar should also include "SQL UDFs" to ensure users are aware that we support it. The item can redirect to https://docs.risingwave.com/docs/current/sql-create-function/#sql-udfs 

(Although I wanted to help, I couldn't find how to make a sidebar item redirect to another page.

### Reference

_No response_

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Info

- **Description**

  - Add a single page of SQL UDF in the sidebar to make it more obvious. But now there is a small issue: we have duplicated content for SQL UDFs on both the "create function" page and the "SQL UDFs" page. Therefore, we need to ensure synchronization between the two when updating SQL UDFs in the future. I haven't come up with a better solution yet. Welcome any suggestions!

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/2270

## For reviewers

- **[SQL UDF*](https://pr-2290.d2fbku9n2b6wde.amplifyapp.com/docs/dev/ql-udfs/)*

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

**Comment by neverchanje** [comment]
> we have duplicated content for SQL UDFs on both the "create function" page and the "SQL UDFs" page. 

My idea was to redirect the link of SQL UDFs to CREATE FUNCTION, or vise versa.

**Review by neverchanje — APPROVED** [review]

File: docs/sql/udf/sql-udfs.md [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+
++ Recursive definition is NOT supported at present. For example, the statement `create function recursive(INT, INT) returns int language sql as 'select recursive($1, $2) + recursive($1, $2)';` will fail.
+
+:::
+
+## Examples
+
+At present, we support SQL UDFs with unnamed and named parameters. This section offers examples of the current supported syntax. We will offer some basic examples first to help you understand and grasp them. Then, we will offer more examples that are closer to real-world scenarios, such as a mock table, for your further practice and understanding.
+
+### Basic examples for SQL UDFs with unnamed and named parameters
```
Comment (WanYixian) [inline docs/sql/udf/sql-udfs.md:45]:
```suggestion
### Basic SQL UDFs with unnamed and named parameters
```

File: docs/sql/commands/sql-create-function.md [hunk 2]
```diff
@@ -103,375 +106,17 @@ For more details, see [Use UDFs in Rust](/sql/udf/udf-rust.md).
 
 ## SQL UDFs
 
-You can also define SQL UDFs in RisingWave by using the `CREATE FUNCTION` command.
+SQL UDFs in RisingWave are designed to expand directly into expressions at the frontend, resulting in minimal performance difference compared to manually calling multiple functions. 
 
-### Syntax
+The `CREATE FUNCTION` command is used to define SQL UDFs. You can read our [specific guide dedicated to SQL UDFs](/sql/udf/sql-udfs.md) for more details.
```
Comment (WanYixian) [inline docs/sql/commands/sql-create-function.md]:
```suggestion
The `CREATE FUNCTION` command is used to define SQL UDFs. You can read our guide on [SQL UDFs](/sql/udf/sql-udfs.md) for more details.
```

**Review by WanYixian — APPROVED** [review]
Others LGTM!

**Pull request merged**